pavelib/tests.py

```diff
@@ -88,8 +89,20 @@
         '--xdist_ip_addresses',
         dest='xdist_ip_addresses',
         help="Comma separated string of ip addresses to shard tests to via xdist."
-    )
-], share_with=['pavelib.utils.test.utils.clean_reports_dir'])
+    ),
+    make_option(
+        '--with-wtw',
+        dest='with_wtw',
+        action='store_true',
+        default=False,
+        help="Only run tests based on the lines changed relative to the specified branch"
+    ),
+    ("compare-branch=", "b", "Branch to compare against, defaults to origin/master"),
```

Comment:
Why not combine these two options?  `--with-wtw=branchname`.  Specifying master explicitly wouldn't be bad.

Reply:
Yeah, that makes sense